app/models/subjects/further_education_subject.rb

```diff
@@ -13,4 +13,7 @@
 #
 
 class FurtherEducationSubject < Subject
+  def self.find
```

Comment:
Shouldn't it be in the same vein as 
https://github.com/DFE-Digital/manage-courses-backend/blob/13949f0b130f2213a5b5cfbb3a944c0b21b86d0e/app/models/subjects/secondary_subject.rb#L16-L18

it seems rather odd to do 

``` ruby
subject = FurtherEducationSubject.find
```

Reply:
Further education is it's own subject - there's only one. I went with this because `FurtherEducation.further_education` looked a bit weird

I thought about doing `Subjects.further_education` on the `Subject` model, IMO `FurtherEducation` should have all the information about `FurtherEducation` subjects, so splitting it into two places seems wrong

Reply:
How about `instance`?

Reply:
Doing a quick search through the Ruby docs to see if this is a reserved keyword (it's not) turns up some prior art of doing it this way.

Reply:
:+1: sounds good to me